pkgs/development/libraries/rocksdb/default.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv
+{ stdenv, lib
```

Comment:
Use `stdenv.lib` instead.

Reply:
@yegortimoshenko  I always write `lib` as a formal argument in my expressions; have we made a formal decision somewhere to prefer `stdenv.lib` over a separate `lib` argument?

Reply:
I've seen other reviewers ask that (@joachifm, @orivej), but I haven't seen a discussion.

Reply:
I'd wager that `stdenv.lib` vastly  outnumbers passing `lib` separately (for package expressions). I'm unaware of any arguments beyond uniformity.

Whether it makes sense for `stdenv` to provide `lib` is debatable I suppose, but it *does* & is widely relied upon, so to me it makes more sense to just stick with convention for convention's sake.